test-resources/project/test-lib/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
```

Comment:
I had added this originally, but then I didn't write tests against it.  Better clean it up.